templates/taxi/car_delete.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %} I
+{% block content %}
+<h1>Delete Car</h1>
+
+<p>Are you sure you want to delete the car: {{ car }}?</p>
+  <form action="" method="post">
+{% csrf_token %}
+<input type="submit" value="Yes, delete" class="btn btn-danger" />
```

Comment:
Add Cancel button apart from Delete one. The Cancel button will lead to the previous page the user was on.